luke/index.js

```diff
@@ -0,0 +1,21 @@
+let chrs = "ABCDEFGH";
+
+// 🚀 Write Program to picking one random character from chrs.
+let pickRandom = Math.floor(Math.random() * chrs.length);
+console.log(chrs[pickRandom]);
+
+let sy = "%& § !* @= ";
```

Comment:
After some tries I get empty value,  there was a typo in repository and Ghassan has fixed so let sy should be like this=>
let sy = " %&§!*@= ";
Means white spaces are at the beginning and at the end, therefore you can use trim() instead of  RegExp

Reply:
You also still can get the same result if you wanna use replace method and RegExp like this:
let sy = "%& § !* @= ";
let alteration = sy.replace(/ /g, "");
let pickRandom2 = alteration[Math.floor(Math.random() * alteration.length)];
console.log(pickRandom2);